tests/fixtures/issues/issue-876.php

```diff
@@ -86,19 +86,19 @@
             'JavaApplets' => true,
             'ActiveXControls' => false,
             'isMobileDevice' => true,
-            'isTablet' => false,
```

Comment:
These two tests are for Android 3.x, which is the Tablet only release of Android, so I just changed the device for these to "general Tablet" instead of "general Mobile Phone (Touch)".